tests/wpt/metadata/html/semantics/forms/constraints/form-validation-validity-valueMissing.html.ini

```diff
@@ -1,323 +1,5 @@
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is -0]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is a number with some white spaces]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is Math.pow(2, 1024)]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is Math.pow(-2, 1024)]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is a string that can not be coverted to a number]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The value attribute is empty string]
-    expected: FAIL
-
-  [[INPUT in CHECKBOX status\] The required attribute is not set]
-    expected: FAIL
-
-  [[INPUT in CHECKBOX status\] The checked attribute is true]
-    expected: FAIL
-
-  [[INPUT in CHECKBOX status\] The checked attribute is false]
-    expected: FAIL
-
-  [[INPUT in RADIO status\] The required attribute is not set]
-    expected: FAIL
-
-  [[INPUT in RADIO status\] The checked attribute is true]
-    expected: FAIL
-
-  [[INPUT in RADIO status\] The checked attribute is false]
-    expected: FAIL
-
-  [[INPUT in FILE status\] The required attribute is not set]
-    expected: FAIL
-
-  [[INPUT in FILE status\] The Files attribute is null]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] Value is a string that cannot be converted to a number]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The required attribute is not set]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Valid local date and time string(2000-12-10T12:00:00)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Valid local date and time string(2000-12-10 12:00)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Valid local date and time string(1979-10-14T12:00:00.001)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value attribute is a number(1234567)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value attribute is a Date object]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Invalid local date and time string(1979-10-99 99:99)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Valid local date and time string(1979-10-14 12:00:00)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] Invalid local date and time string(2001-12-21  12:00)-two white space]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] the value attribute is a string(abc)]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value attribute is empty string]
-    expected: FAIL
-
-  [[textarea\]  The value is not empty]
-    expected: FAIL
-
-  [[textarea\]  The value is empty]
-    expected: FAIL
-
-  [[textarea\]  The required attribute is not set]
-    expected: FAIL
-
-  [[select\]  Selected the option with value equals to empty]
-    expected: FAIL
-
   [[INPUT in RADIO status\] The checked attribute is false and the name attribute is empty]
     expected: FAIL
```

Comment:
It is radio group membership issue, see https://github.com/whatwg/html/issues/5202. Spec doesn't specify that nameless elements don't have radio button groups.